initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -559,26 +559,24 @@ stop_component() {
 
     eval $stopcmd
 
+    unlock ${LOCKPATH}
+    removePid ${PIDPATH}
+
     RESULT=0
-    local counter=0
-    while [[ $RESULT -eq 0 && $counter -le 20 ]]; do
+    local waittime=30
```

Comment:
I can tell what compType is being run, and if dali increase it to a more reasonable value.  What would you suggest for real world environments?